python/paddle/autograd/autograd.py

```diff
@@ -64,7 +68,12 @@ class Jacobian:
 
     """
 
-    def __init__(self, ys, xs, is_batched=False):
+    def __init__(
+        self,
+        ys: Tensor | tuple[Tensor, ...],
+        xs: Tensor | tuple[Tensor, ...],
```

Comment:
```suggestion
        ys: Tensor
        xs: Tensor,
```

文档应该有问题，不能是 `tuple` ，不然后面 `ys.shape` 会出错 ~

Reply:
收到